AndroidHomeApplication/app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,306 @@
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="4dp"
+            android:text="@string/action_adventure_fantasy"
+            android:textColor="@color/radical_red_darker"
+            app:layout_constraintEnd_toStartOf="@+id/guideline_end"
+            app:layout_constraintStart_toEndOf="@+id/guideline_start"
+            app:layout_constraintTop_toBottomOf="@+id/text_title" />
+
+        <ImageView
+            android:id="@+id/star_1"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            app:layout_constraintStart_toEndOf="@+id/guideline_start"
+            app:layout_constraintBottom_toBottomOf="@+id/star_2"
+            app:layout_constraintTop_toTopOf="@+id/star_2"
+            app:srcCompat="@drawable/ic_star"
+            app:tint="@color/radical_red" />
+
+        <ImageView
+            android:id="@+id/star_2"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="4dp"
+            app:layout_constraintBottom_toBottomOf="@+id/star_3"
+            app:layout_constraintStart_toEndOf="@+id/star_1"
+            app:layout_constraintTop_toTopOf="@+id/star_3"
+            app:srcCompat="@drawable/ic_star"
+            app:tint="@color/radical_red" />
+
+        <ImageView
+            android:id="@+id/star_3"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="4dp"
+            app:layout_constraintBottom_toBottomOf="@+id/star_4"
+            app:layout_constraintStart_toEndOf="@+id/star_2"
+            app:layout_constraintTop_toTopOf="@+id/star_4"
+            app:srcCompat="@drawable/ic_star"
+            app:tint="@color/radical_red" />
+
+        <ImageView
+            android:id="@+id/star_4"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="4dp"
+            app:layout_constraintBottom_toBottomOf="@+id/star_5"
+            app:layout_constraintStart_toEndOf="@+id/star_3"
+            app:layout_constraintTop_toTopOf="@+id/star_5"
+            app:srcCompat="@drawable/ic_star"
+            app:tint="@color/radical_red" />
+
+        <ImageView
+            android:id="@+id/star_5"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="4dp"
+            app:layout_constraintBottom_toBottomOf="@+id/text_reviews"
+            app:layout_constraintStart_toEndOf="@+id/star_4"
+            app:layout_constraintTop_toTopOf="@+id/text_reviews"
+            app:srcCompat="@drawable/ic_star"
+            app:tint="@color/storm_gray" />
+
+        <TextView
+            android:id="@+id/text_reviews"
+            android:layout_width="0dp"
```

Comment:
android:layout_width="0dp"
 app:layout_constraintStart_toEndOf="@+id/star_5"
 
 that's all?